lib/Dialect/Torch/Transforms/ShapeLibrary.cpp

```diff
@@ -6980,6 +6980,10 @@ module {
     %2 = call @__torch__.torch.jit._shape_functions.sum_mean_dim(%arg0, %0, %arg2, %1) : (!torch.list<int>, !torch.optional<list<int>>, !torch.bool, !torch.any) -> !torch.list<int>
     return %2 : !torch.list<int>
   }
+  func.func @"__torch_mlir_shape_fn._torch_mlir_custom_op_example.identity"(%arg0: !torch.list<int>) -> !torch.list<int> {
+    %0 = call @__torch__.torch.jit._shape_functions.unary(%arg0) : (!torch.list<int>) -> !torch.list<int>
+    return %0 : !torch.list<int>
+  }
```

Comment:
Will this fail if we check this in but someone builds with custom_op_example ?

Reply:
I think you meant what would happen when someone doesn't build custom_op_example? It would not cause any error